server/selinux/server/relabel.sh

```diff
@@ -1,13 +1,20 @@
 #!/bin/sh
 
-/sbin/restorecon -i -R /etc/httpd/conf.d/pulp.conf
-/sbin/restorecon -i -R /etc/pulp
-/sbin/restorecon -i -R /etc/pki/pulp
-/sbin/restorecon -i -R /srv/pulp
-/sbin/restorecon -i -R /usr/bin/pulp-admin
-/sbin/restorecon -i -R /usr/bin/pulp-consumer
-/sbin/restorecon -i -R /var/lib/pulp
-/sbin/restorecon -i -R /var/log/pulp
-
-# Relabel the celery binary
-/sbin/restorecon -i -R /usr/bin/celery
+# If upgrading from before 2.4.0
```

Comment:
if you save this little bit as an executable you can try this for yourself. http://fpaste.org/201685/14271384/